pages/docs/guides/pause-functions.mdx

```diff
@@ -0,0 +1,52 @@
+export const description = 'Learn how to pause an Inngest function.'
+
+# Pausing functions
+
+Inngest allows you to pause a function indefinitely. This is a powerful feature that can be useful if you are planning downtime for a database migration, for example, or if a user reports a bug and you want to stop processing events until you've fixed it.
```

Comment:
```suggestion
Inngest allows you to pause a function indefinitely. This is a powerful feature that can be useful, for example, if you are planning downtime for a database migration or if a user reports a bug and you want to stop processing events until you've fixed it.
```

the mid-sentence "for example" sounds conversational

Reply:
Also: _technically_, the function can be paused indefinitely, but does the plan tier impact it?

Reply:
✅ Clarified in 5feaae19a167fb694d2e3e0d5cf373d30be01c19.